csrc/compute_at_map.cpp

```diff
@@ -715,6 +716,13 @@ void IterDomainGraph::build(Fusion* fusion) {
     if (auto merge = dynamic_cast<Merge*>(def)) {
       if (merge->inner()->extent()->isOneInt()) {
         almost_exact_nodes_.mapEntries(merge->outer(), merge->out());
+        permissive_relaxed_resize_nodes_.mapEntries(
+            merge->outer(), merge->out());
+      } else {
+        // maps to inner dimension, even though it's not an identical mapping.
```

Comment:
Now I see why we want to have this map, which is effectively used to traverse innermost IDs.

I wonder if there's some overlap with `getInnerLeafDim`, which also does a similar innermost traversal. Can it be simplified with this map? Or can it replace this map?

Reply:
Right now these two aren't interchangeable, `getInnerLeafDim` requires an exact mapping on leaf domain, since the transpose scheduler expects that for tiling. While `permissive_relaxed_resize_nodes_` blindly maps IDs to its `inner` part across split/merge.